<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconSelectWidget</class>
 <widget class="QWidget" name="IconSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>182</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>39</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="pIconBtn3">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Icon/Settings/Head3.png</normaloff>
         <activeoff>:/Icon/Settings/Head3_Select.png</activeoff>
         <activeon>:/Icon/Settings/Head3_Select.png</activeon>
         <selectedoff>:/Icon/Settings/Head3_Select.png</selectedoff>
         <selectedon>:/Icon/Settings/Head3_Select.png</selectedon>:/Icon/Settings/Head3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pIconBtn4">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Icon/Settings/Head4.png</normaloff>
         <normalon>:/Icon/Settings/Head4.png</normalon>
         <activeoff>:/Icon/Settings/Head4_Select.png</activeoff>
         <activeon>:/Icon/Settings/Head4_Select.png</activeon>
         <selectedoff>:/Icon/Settings/Head4_Select.png</selectedoff>
         <selectedon>:/Icon/Settings/Head4_Select.png</selectedon>:/Icon/Settings/Head4.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="MaskLabel" name="pUploadIconLb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pUploadBtn">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>自定义...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pIconBtn2">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Icon/Settings/Head2.png</normaloff>
         <normalon>:/Icon/Settings/Head2.png</normalon>
         <activeoff>:/Icon/Settings/Head2_Select.png</activeoff>
         <activeon>:/Icon/Settings/Head2_Select.png</activeon>
         <selectedoff>:/Icon/Settings/Head2_Select.png</selectedoff>
         <selectedon>:/Icon/Settings/Head2_Select.png</selectedon>:/Icon/Settings/Head2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pIconBtn1">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/Icon/Settings/Head1.png</normaloff>
         <normalon>:/Icon/Settings/Head1.png</normalon>
         <activeoff>:/Icon/Settings/Head1_Select.png</activeoff>
         <activeon>:/Icon/Settings/Head1_Select.png</activeon>
         <selectedoff>:/Icon/Settings/Head1_Select.png</selectedoff>
         <selectedon>:/Icon/Settings/Head1_Select.png</selectedon>:/Icon/Settings/Head1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>82</width>
         <height>82</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaskLabel</class>
   <extends>QLabel</extends>
   <header>masklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
